<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>rsi_login</class>
 <widget class="QDialog" name="rsi_login">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>402</width>
    <height>246</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>rsi_login</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>10</y>
     <width>341</width>
     <height>191</height>
    </rect>
   </property>
   <property name="title">
    <string>Log In...</string>
   </property>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>40</y>
      <width>278</width>
      <height>131</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="1">
      <widget class="QLineEdit" name="userID_Line_Edit"/>
     </item>
     <item row="3" column="1">
      <widget class="QPushButton" name="logIn_PushButton">
       <property name="text">
        <string>Log In</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Password</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QCheckBox" name="rememberMe_CheckBox">
       <property name="text">
        <string>Remeber Me</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="password_LineEdit">
       <property name="echoMode">
        <enum>QLineEdit::Password</enum>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>User ID</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QLabel" name="sql_db_connection_label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>220</y>
     <width>311</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>TEXT</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>userID_Line_Edit</tabstop>
  <tabstop>password_LineEdit</tabstop>
  <tabstop>rememberMe_CheckBox</tabstop>
  <tabstop>logIn_PushButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
